### PR TITLE
fix: add extra stack outputs and inputs

### PR DESCRIPTION
**Pull request by akocbek**
### Description

issues: 
- https://github.com/terraform-ibm-modules/stack-retrieval-augmented-generation/issues/210
- https://github.com/terraform-ibm-modules/stack-retrieval-augmented-generation/issues/212
- https://github.com/terraform-ibm-modules/stack-retrieval-augmented-generation/issues/203

### Release required?

- [ ] No release
- [x] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by daniel-butler-irl — APPROVED** [review]
Stack defs look good, you might want someone else to take a look at the diagram, I cannot comment if is correct or not

File: solutions/basic/stack_definition.json [hunk 1]
```diff
@@ -747,35 +751,171 @@
...
+      "value": "ref:../members/Essential Security - Logging Monitoring Activity Tracker/outputs/cloud_monitoring_guid"
+    },
+    {
+      "name": "log_analysis_crn",
+      "value": "ref:../members/Essential Security - Logging Monitoring Activity Tracker/outputs/log_analysis_crn"
+    },
+    {
+      "name": "log_analysis_guid",
+      "value": "ref:../members/Essential Security - Logging Monitoring Activity Tracker/outputs/log_analysis_guid"
+    },
```
Comment (ocofaigh) [inline solutions/basic/stack_definition.json:831]: remove `log_analysis_crn` and `log_analysis_guid` - the stack no longer deploys log analysis

File: solutions/standard/stack_definition.json [hunk 2]
```diff
@@ -801,35 +805,187 @@
...
+      "name": "cloud_monitoring_guid",
+      "value": "ref:../members/Essential Security - Logging Monitoring Activity Tracker/outputs/cloud_monitoring_guid"
+    },
+    {
+      "name": "log_analysis_crn",
+      "value": "ref:../members/Essential Security - Logging Monitoring Activity Tracker/outputs/log_analysis_crn"
+    },
+    {
+      "name": "log_analysis_guid",
+      "value": "ref:../members/Essential Security - Logging Monitoring Activity Tracker/outputs/log_analysis_guid"
```
Comment (ocofaigh) [inline solutions/standard/stack_definition.json:884]: remove `log_analysis_crn` and `log_analysis_guid` - the stack no longer deploys log analysis

**Review by ocofaigh — CHANGES_REQUESTED** [review]
@akocbek see comments

**Comment by ocofaigh** [comment]
The pipeline is failing due to known issues with projects race condition. I'm doing a manual test using your branch, and if the outputs look good, Ill proceed to force merge this

**Pull request merged**